nexus/db-queries/src/db/queries/region_allocation.rs

```diff
@@ -67,6 +67,18 @@ type SelectableSql<T> = <
     <T as diesel::Selectable<Pg>>::SelectExpression as diesel::Expression
 >::SqlType;
 
+/// Parameters for the region(s) being allocated
+#[derive(Debug, Clone, Copy)]
+pub struct RegionParameters {
+    pub block_size: u64,
+    pub blocks_per_extent: u64,
+    pub extent_count: u64,
+
+    /// True if the region will be filled with a Clone operation and is meant to
+    /// be read-only.
```

Comment:
Ah, so we do create a region with this path for the clone to fill up.

Reply:
:+1: